### ISSUE TITLE
Need guide on what the .devcontainer/ and all the files inside infra/ folder does

### ISSUE THREAD

**Issue by msr-wtag** [jasontaylordev/CleanArchitecture#1237]
cannot understand why i need those and where to use

**Comment by jasontaylordev** [comment]
Yes, sorry about that; I'll update the documentation shortly. In essence the template is now compatible with the Azure Developer CLI. You can deploy it to Azure using `azd up` and you can learn all about `azd` here; https://aka.ms/awesome-azd.
